src/components/tx/SignOrExecuteForm/index.tsx

```diff
@@ -164,7 +164,9 @@ const SignOrExecuteForm = ({
     setAdvancedParams(data)
   }
 
-  const submitDisabled = !isSubmittable || isEstimating || !tx
+  const walletIsWrongChain = currentChain?.chainId != wallet?.chainId
```

Comment:
We've just merged another PR that adds `const isWrongChain = useIsWrongChain()`. Please pull from dev, you can then use this `isWrongChain` var.